rpcs3/rpcs3qt/fatal_error_dialog.cpp

```diff
@@ -4,6 +4,17 @@
 #include <QTextDocument>
 #include <QIcon>
 
+static QString process_dialog_text(std::string_view text)
+{
+	auto html = Qt::convertFromPlainText(QString::fromUtf8(text.data(), text.size()));
+
+	// Let's preserve some html elements destroyed by convertFromPlainText
+	const QRegExp link_re{ R"(&lt;a\shref='([a-z0-9?=&#:\/\.\-]+)'&gt;([a-z0-9?=&#:\/\.\-]+)&lt;\/a&gt;)", Qt::CaseSensitive, QRegExp::RegExp2};
```

Comment:
Kinda confused why this is necessary

Reply:
Because convertFromPlainText will destroy any html elements in the input text. Since we're providing the user with links (i.e <a href/> elements) we need a regex to restore the original. It was either that or magic tokens which I don't like. Or plaintext links that the user has to copy manually and paste which is bad user experience.
And removing convertFromPlainText means \n is not converted to <br> and so on.
There are other ways, but this was fairly easy to set up to preserve links.

Reply:
I think it would be simpler to just add a param (bool is_html = false) and just skip the convertFromPlainText if it's true.

Reply:
![image](https://user-images.githubusercontent.com/23019877/205757843-91e37811-29b2-4e55-8e0d-5356c39f6ec2.png)

Reply:
if you do that, you need to extract the style from the outer paragraph though so it is used in all internal paragraphs:
![image](https://user-images.githubusercontent.com/23019877/205758922-999cd40d-0fc4-469e-84d2-4a1204e54b93.png)

Reply:
In case you wanna do this, see https://github.com/kd-11/rpcs3/pull/56
(haven't tested it with the actual dlls)

Reply:
:( sensible designs are no fun. Complex regexes ftw XD

Reply:
The issue with your "solution" is that it will print html when the error text from the source is supposed to be printed raw